core/sr-primitives/src/lib.rs

```diff
@@ -272,7 +272,7 @@ impl From<codec::Compact<Permill>> for Permill {
 
 /// Perbill is parts-per-billion. It stores a value between 0 and 1 in fixed point and
 /// provides a means to multiply some other value by that.
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug, Ord, PartialOrd))]
```

Comment:
why only `Ord` in `std`? doesn't wasm need it to?

Reply:
It must be used only in tests but will double check.

Reply:
`Fixed64` has it in both cases; might be worth making consistent.